src/ansys/mapdl/core/launcher.py

```diff
@@ -947,7 +948,7 @@ def launch_mapdl(
     replace_env_vars=None,
     version=None,
     **kwargs,
-) -> _MapdlCore:
+) -> MapdlGrpc:
```

Comment:
It can return that, or an MapdlCorba class or MapdlConsole class. But having those there, force me to import those clases, and CORBA is only compatible on python 3.8 or lower.

Reply:
Any idea how to solve that?

Reply:
> It can return that, or an MapdlCorba class or MapdlConsole class. But having those there, force me to import those clases, and CORBA is only compatible on python 3.8 or lower.

Are you sure about that ? From what I can see you can only return a MapdlGrpc

However if that's not the case you can still import these class under a `if TYPE_CHECKING`:
```python
from typing import TYPE_CHECKING
import sys

if TYPE_CHECKING:
    from ansys.mapdl.core.mapdl_console import MapdlConsole
    # Here we can find a way to check for python version
        from ansys.mapdl.core.mapdl_corba import MapdlCorba
```

Reply:
My bad, indeed you can return a MapdlCorba or MapdlCOnsole